_layouts/post.html

```diff
@@ -0,0 +1,147 @@
+---
+layout: default
+---
+
+<!--================Forum Breadcrumb Area =================-->
+<section class="doc_banner_area search-banner-light" style='padding:50px 0'>
+</section>
+<section class="page_breadcrumb">
+  <div class="container custom_container">
+    <div class="row">
+      <div class="col-sm-7">
+        <nav aria-label="breadcrumb">
+          <ol class="breadcrumb">
+            <li class="breadcrumb-item"><a href="https://www.lovelace.academy">Home</a></li>
+            <li class="breadcrumb-item"><a href="https://learn.lovelace.academy">{{ page.categories[0] }}</a></li>
+            <li class="breadcrumb-item active" aria-current="page">{{ page.title }}</li>
+          </ol>
+        </nav>
+      </div>
+      <div class="col-sm-5">
+        <a href="#" class="date"><i class="icon_clock_alt"></i>Updated on {{ page.last_modified_at | default: page.date | date: '%b %d, %y' }}</a>
```

Comment:
Small one, can we make the year component in the yyyy format?

Reply:
Done